pkg/reconciler/v1alpha1/clusteringress/config/istio.go

```diff
@@ -40,11 +41,11 @@ const (
 var (
 	defaultGateway = Gateway{
 		GatewayName: "knative-ingress-gateway",
-		ServiceURL:  "istio-ingressgateway.istio-system.svc.cluster.local",
+		ServiceURL:  fmt.Sprintf("istio-ingressgateway.istio-system.svc.%s", utils.GetClusterDomainName()),
```

Comment:
I would use string concat (`+`) rather than Sprintf here, or `strings.Join(...., ".")`.

Reply:
hm, right concat should be more efficient than sprintf, will change it. Thanks for suggestion.